src/helpers/getNativeNamesValues.ts

```diff
@@ -0,0 +1,13 @@
+import { Name, Translation } from '../types/CountriesTypes';
+
+export const getNativeNamesValues = (name:Name) => {
+  const nativeNamesKeys = Object.keys(name?.nativeName || {});
+
+  const nativeNamesValues:Translation[] = nativeNamesKeys.map(key => {
+    return name?.nativeName ? name.nativeName[key] : { official: "", common: "" }
```

Comment:
Más que retornar un objeto con propiedades vacías, retorna las propiedades con su respectivo valor, aunque no estoy seguro si utilizas realmente ese objeto cuando no existe un `nativeName`:

```suggestion
    return name?.nativeName ? name.nativeName[key] : { official: name.official, common: name.common }
```